lib/open-pgp-key-pair.ts

```diff
@@ -81,39 +115,46 @@ export class OpenPGPKeyPair extends cdk.Construct implements ICredentialPair {
       description: 'Generates an OpenPGP Key and stores the private key in Secrets Manager and the public key in an SSM Parameter',
       code: new lambda.AssetCode(codeLocation),
       handler: 'index.handler',
-      timeout: 300,
-      runtime: lambda.Runtime.NodeJS810,
+      timeout: cdk.Duration.seconds(300),
+      runtime: lambda.Runtime.NODEJS_8_10,
```

Comment:
Should we use 10.x?

Reply:
I'd rather leave this as-is, if you don't mind. I think this should be a separate change, and we still have some time to use Node 8 in Lambda.